wp/plugin/wp.ml

```diff
@@ -106,7 +106,7 @@ let analyze_proj (proj : project) (var_gen : Env.var_gen) (ctx : Z3.context)
   let pre, env' = Pre.visit_sub env post main_sub in
   Format.printf "\nSub:\n%s\nPre:\n%a\n%!"
     (Sub.to_string main_sub) Constr.pp_constr pre;
-  (pre, env', env)
+  (pre, env, env')
 
```

Comment:
I'm a bit disturbed by this bug. Is there any way we can return more structured output to avoid these kinds of shenanigans in the future? This is probably a task for myself.